Alignment/OfflineValidation/plugins/TrackerGeometryCompare.cc

```diff
@@ -19,6 +19,7 @@
 #include "CondFormats/DataRecord/interface/SiPixelQualityRcd.h"
 #include "CalibFormats/SiStripObjects/interface/SiStripQuality.h"
 #include "CalibTracker/Records/interface/SiStripQualityRcd.h"
+#include "CondCore/SiPixelPlugins/interface/PixelRegionContainers.h"
```

Comment:
```suggestion
#include "CondCore/SiPixelPlugins/interface/SiPixelPayloadInspectorHelper.h"
```

I guess you want actually this, to have access to 

https://github.com/cms-sw/cmssw/blob/eb6caadb0fadad053b08cd563264cd7e2370f177/CondCore/SiPixelPlugins/interface/SiPixelPayloadInspectorHelper.h#L39

Reply:
true, it was only inherited through PixelRegionContainers it seems